Assets/Prefabs/UI/OfficeTerminal/Missions/MissionsPanel.prefab

```diff
@@ -33,7 +33,7 @@ RectTransform:
   m_RootOrder: 0
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
   m_AnchorMin: {x: 0, y: 0}
-  m_AnchorMax: {x: 0, y: 0}
```

Comment:
These prefabs, I swear the positioning I didn't touch. At least they look the same in-game. I added reference to the new script so needed to commit these bastards.